hudi-io/src/test/java/org/apache/hudi/io/storage/TestHoodieStorageBase.java

```diff
@@ -148,6 +150,46 @@ public void testCreateWriteAndRead() throws IOException {
     assertTrue(storage.createDirectory(path4));
   }
 
+  @Test
+  public void testSeekable() throws IOException {
+    HoodieStorage storage = getHoodieStorage();
+    StoragePath path = new StoragePath(getTempDir(), "testSeekable/1.file");
+    assertFalse(storage.exists(path));
+    byte[] data = new byte[] {2, 42, 49, (byte) 158, (byte) 233, 66, 9, 34, 79};
```

Comment:
why did you need to cast the values over 100?

Reply:
Because for values larger than 127 it has to cast from int to byte (or have to use negative value here, which is not going to be readable).